inc/Engine/Preload/FormatUrlTrait.php

```diff
@@ -8,18 +8,22 @@ trait FormatUrlTrait {
 	 * Format URL.
 	 *
 	 * @param string $url URL.
-	 *
+	 * @param bool   $use_website_trailing Use the website config for trailing slash.
 	 * @return string
 	 */
-	public function format_url( string $url ): string {
+	public function format_url( string $url, bool $use_website_trailing = false ): string {
 		$queries = wp_parse_url( $url, PHP_URL_QUERY ) ?: '';
 		$queries = $this->convert_query_to_array( $queries );
 
 		ksort( $queries );
 
 		$url = strtok( $url, '?' );
 
-		$url = untrailingslashit( $url );
+		if ( $use_website_trailing ) {
+			$url = user_trailingslashit( $url );
+		} else {
+			$url = untrailingslashit( $url );
+		}
```

Comment:
`user_trailingslashit()` removes the trailing slash if there is one and shouldn't, and add one if there isn't and it should. So we don't need the condition and can use this function directly.

Reply:
missed this comment?